cob_moveit_bringup/launch/chomp_planning_pipeline.xml

```diff
@@ -0,0 +1,12 @@
+<launch>
+
+   <!-- CHOMP Plugin for MoveIt! -->
+   <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+   <arg name="planning_plugin" value="chomp_interface/CHOMPPlanner" />
```

Comment:
using `chomp_interface` would require to define an `<exec_depend>` to `chomp_interface`

@ipa-bfb 
is this package installable from debs?